Makefile

```diff
@@ -0,0 +1,86 @@
+VERSION := 0.19.1
+
+# install directory layout
+PREFIX ?= /usr/local
+INCLUDEDIR ?= $(PREFIX)/include
+LIBDIR ?= $(PREFIX)/lib
+PCLIBDIR ?= $(LIBDIR)/pkgconfig
+
+# collect C++ sources, and link if necessary
+CPPSRC := $(wildcard src/*.cc)
+
+ifeq (, $(CPPSRC))
+	ADDITIONALLIBS := 
+else
+	ADDITIONALLIBS := -lc++
+endif
+
+# collect sources
+ifneq ($(AMALGAMATED),1)
```

Comment:
I think we don't need an `AMALGAMATED` option here, so I would just make this part unconditional, and remove the `else` part.

Reply:
And there is never a `lib.c`, so we don't need the `filter-out` part either.

Reply:
I attempted to do some research on this, but wasn't 100% sure what was going on. Removing it sounds good to me.